.editorconfig

```diff
@@ -0,0 +1,6 @@
+root = true
+
+[**]
+indent_style = space
+indent_size = 2
+insert_final_newline = true
```

Comment:
Sadly, VS2017 has default support for editor config but ignores the 2 most useful options: insert_final_newline and trim_trailing_whitespace